src/libstd/time.rs

```diff
@@ -245,15 +245,15 @@ impl Instant {
     /// Returns `Some(t)` where `t` is the time `self + duration` if `t` can be represented as
     /// `Instant` (which means it's inside the bounds of the underlying data structure), `None`
     /// otherwise.
-    #[unstable(feature = "time_checked_add", issue = "55940")]
+    #[stable(feature = "time_checked_add", since = "1.34.0")]
     pub fn checked_add(&self, duration: Duration) -> Option<Instant> {
         self.0.checked_add_duration(&duration).map(|t| Instant(t))
```

Comment:
Not part of your changes... However, looking at these lambdas (in all 4 methods), they look redundant; so can you change it to?:
```suggestion
        self.0.checked_add_duration(&duration).map(Instant)
```

Reply:
Sure. Good catch. I'm pushing that soon.